new/components/amqpinflux/scripts/create.sh

```diff
@@ -29,7 +29,7 @@ yum_install ${AMQPINFLUX_RPM_SOURCE_URL}
 
 create_service_user ${AMQPINFLUX_USER} ${AMQPINFLUX_HOME}
 
-if [[ "${RABBITMQ_CERT_ENABLED}" == 'True' ]]; then
+if [[ "${RABBITMQ_CERT_ENABLED}" == 'true' ]]; then
```

Comment:
It does work- tested.

The 'ctx -j node properties xyz' retrieves the named property as the json value of true (lower case). If not using '-j' then we get the python version but nothing when it's false.